### ISSUE TITLE
Recommendation

### ISSUE THREAD

**Issue by FAXES** [Itz-Hyperz/faxstore-api-v1#1]
Instead of having it run as it's own app, you could build it into FaxStore itself, that way it's only a drag and drop for the user.

You would do this by calling an extra file via FaxStores config file, when these js files get executed the `app` (express) and `connection` (SQL) is passed to the file, then you just build off from inside that file, saves multiple connections to the database from different apps, and also prevents it being another whole application to run.

Example:
```js
const config = require('./config.json');

module.exports = function(app) {
    console.log(`API started`);
    var apiSecret = "FAXESYes";

    app.get('/api/partners', async function (req, res) {
        connection.query(`SELECT * FROM partners`, async (err, results) => {
             res.json(results)
        });
    });
}
```

This would simplify and make the installation for users easier before a whole API is built in.

**Comment by Itz-Hyperz** [comment]
what is passed to the files, like, is it both `app` and `connection`?

**Comment by FAXES** [comment]
Yes, both are passed through to the extension files. Future plans to also move Discord bot object etc

**Comment by Itz-Hyperz** [comment]
I will likely push that in the near future.

**Comment by braxtongpoll** [comment]
Faxes is right, I use this way as well.

**Comment by braxtongpoll** [comment]
https://pluto.vodka/plutospics09276.png
